internal/pkg/levee/levee_test.go

```diff
@@ -32,7 +32,7 @@ func TestLevee(t *testing.T) {
 	if *debugging {
 		Analyzer.Requires = append(Analyzer.Requires, debug.Analyzer)
 	}
-	analysistest.Run(t, dataDir, Analyzer, "./...")
+	analysistest.Run(t, dataDir, Analyzer, "./src/example.com/...")
```

Comment:
This is needed because running on files under `nopanic.com` will produce a "no diagnostic was reported" error.